assets/scripts/temp.js

```diff
@@ -0,0 +1,28 @@
+var options = {
+    enableHighAccuracy: true,
+    timeout: 5000,
+    maximumAge: 0
+};
+
+function success(userPosition){
+    let userCoordinatesResponseObject = userPosition.coords;
+    let userCoordinatesObject = {
+        usersLatitude:userCoordinatesResponseObject.latitude,

```

Comment:
This file looks good -
Only suggestion is a semantic one - you could simplify the usersLatitude/usersLongittude to just usersLat/usersLon

Reply:
I will remove this file all together as this entire file has been deprecated